srcopsmetrics/cli.py

```diff
@@ -98,6 +99,12 @@ def get_entities_as_list(entities_raw: Optional[str]) -> List[str]:
             are stored. Default knowledge path is {StoragePath.DEFAULT.value}
             """,
 )
+@click.option(
+    "--metrics", "-m", is_flag=True, required=False, help=f"""Launch Metrics Calculation for specified repository.""",
+)
+@click.option(
+    "--kebechet", "-K", is_flag=True, required=False, help=f"""Launch Metrics Calculation for specified repository.""",
```

Comment:
also the option maybe should be called `--thoth` as these are thoth related metrics

Reply:
renamed, now present as  `--thoth` or `-T`